airflow/models.py

```diff
@@ -28,12 +29,15 @@
 from airflow.executors import DEFAULT_EXECUTOR, LocalExecutor
 from airflow.configuration import conf
 from airflow.utils import (
-    AirflowException, State, apply_defaults, provide_session)
+    AirflowException, State, apply_defaults, provide_session,
+    is_container, as_tuple)
 
 Base = declarative_base()
 ID_LEN = 250
 SQL_ALCHEMY_CONN = conf.get('core', 'SQL_ALCHEMY_CONN')
 DAGS_FOLDER = os.path.expanduser(conf.get('core', 'DAGS_FOLDER'))
+RETURN_XCOM = '<Returned XCom>'
```

Comment:
I'm fine with `return_value` -- my thinking with the special characters was to minimize the chance of it overwriting any user-created XCom keys.